tomaty/lib/utilities.py

```diff
@@ -0,0 +1,5 @@
+from datetime import timedelta, datetime
```

Comment:
this is simply a ?: is making a lib file like this standard practice? if so, should it stay as is or would merging it with the serialization module be a good idea? I only ask because serialization seems like a util type of role.

Reply:
Having a spot to put methods or classes that serve a general purpose or don't quite fit in with the meat of the application isn't uncommon at all- kind of like how you don't reimplement sqrt in the middle of whatever you're doing, but use math.sqrt instead.

Reply:
It's absolutely worth having a /lib/utilities/date_utilities.py and perhaps a lib/serialization/tomaty_serialization.py, where /lib/utilities are any one off modules and methods that don't fit in with the rest of the application.

Reply:
k!